pkg/rpc/server/testdata/test_contract.go

```diff
@@ -86,6 +90,16 @@ func TotalSupply() int {
 	return totalSupply
 }
 
+func OnNEP17Payment(from interop.Hash160, amount int, data interface{}) {
```

Comment:
Why this one? This contract does not receive/manage assets. But it _must_ call `onNEP17Payment` in `Transfer` to be NEP-17 compliant.

Reply:
I misunderstood you, fixed now.